### PR TITLE
vct-18 addad type & title-string  done

### PR DESCRIPTION
**Pull request by DmitriyDemyanov**

### CODE REVIEW

File: src/components/widget-item/widget-item.component.jsx [hunk 1]
```diff
@@ -16,13 +16,25 @@ const WidgetItem = ({ item }) => {
...
+    };
+    const dataAttention = {
+        stringTitle: `${title} widget already added`,
+        type: 'attention'
+    };
+
     const addWidgetToStore = (item) => {
         if (linkCorrection.includes(item.link)) {
             return nav(link);
         }
```
Comment (khlepitko-gennady) [inline src/components/widget-item/widget-item.component.jsx:31]: You don't need to invoke "find" method twice.

File: src/store/modal/modal.reducer.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+  isShow: false,
+  type: null,
+
 }
 
 export const modalReducer = (state = INITIAL_STATE,action) => {
   const { type,payload } = action;
   switch (type) {
-    case MODAL_ACTIVE_TYPES.SET_MODAL_TITLE:
+    case MODAL_ACTIVE_TYPES.SET_MODAL_SETTINGS:
```
Comment (khlepitko-gennady) [inline src/store/modal/modal.reducer.js:13]: Dev logs must be removed

File: src/store/modal/modal.action.js [hunk 3]
```diff
@@ -1,13 +1,14 @@
 import { MODAL_ACTIVE_TYPES } from "./modal.types";
```
Comment (khlepitko-gennady) [inline src/store/modal/modal.action.js:2]: Wrong naming.
You set up not only "title", but "type" also. Function for these actions named as "setModalTitle", so it means it should handle only a title.
You need to rename this function. Its name's must describe what it doing. "setModalData" or "setModalSettings" e.t.c.

File: src/store/widgets/widgets.reducer.js [hunk 4]
```diff
@@ -66,39 +66,39 @@ const INITIAL_STATE = {
```
Comment (khlepitko-gennady) [inline src/store/widgets/widgets.reducer.js:66]: You can change it in one string in scss file, simply aplly "text-transform: capitalize"

File: src/components/modal-common/modal-common.component.jsx [hunk 5]
```diff
@@ -10,12 +10,20 @@ import { ModalCommonContainer,WrapperRenderModal } from "./modal-common.styles";
...
+  door: 'door-icon',
+  message: 'message-item-icon',
+  attention: 'attention-icon',
+  info: 'modal-default-icon'
+};
+
 const ModalCommon = () => {
   const dispatch = useDispatch();
-  const title = useSelector(modalTitle);
+  const modalItem = useSelector(modalSettings);
```
Comment (khlepitko-gennady) [inline src/components/modal-common/modal-common.component.jsx]: Rename the variable to something more descriptive - settings, modalSettings, modalData or something like that

**Review by khlepitko-gennady — CHANGES_REQUESTED** [review]

File: src/store/widgets/widgets.reducer.js (hunk 4 above)
Comment (DmitriyDemyanov) [reply src/store/widgets/widgets.reducer.js:66]: if I apply  text-transform: capitalize, then all words are converted to upper case.

Comment (khlepitko-gennady) [reply src/store/widgets/widgets.reducer.js:66]: Make sense ))

**Review by khlepitko-gennady — APPROVED** [review]

**Pull request merged**